enacts/flex_fcst/maproom.py

```diff
@@ -634,9 +660,13 @@ def fcst_tiles(tz, tx, ty, proba, variable, percentile, threshold, start_date, l
     # Reading
     
     if CONFIG["forecast_mu_file_pattern"] is None:
-        fcst_mu, fcst_var, obs = cpt.read_pycptv2dataset(DATA_PATH)
+        fcst_mu, fcst_var, obs = cpt.read_mpycptv2dataset(DATA_PATH)
         fcst_mu = fcst_mu.sel(S=start_date)
         fcst_var = fcst_var.sel(S=start_date)
+        if "L" in fcst_mu.dims:
+            fcst_mu = fcst_mu.sel(L=int(lead_time))
+            fcst_var = fcst_var.sel(L=int(lead_time))
+        obs = obs.where(obs["T"].dt.month == fcst_mu["T"].dt.month, drop=True)
         is_y_transform = False
     else:
         fcst_mu, fcst_var, obs, hcst = read_cptdataset(lead_time, start_date, y_transform=CONFIG["y_transform"])
```

Comment:
You didn't change this to `int(lead_time)`. Does that mean `read_cptdataset` expects a string, not an int? That doesn't seem right.

Reply:
That sounds likely but read_cptdataset is for the .tsv case that has many other problems we want to resolve so let's leave that to another PR.